VueJS/App.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div id="app">
+    <Header></Header>
+    <AddTodo v-on:addTodo="addTodo"></AddTodo>
+    <Todos v-bind:todos="todos" v-on:del-todo="deleteTodo" v-on:edit-todo="editTodo"/>
```

Comment:
```suggestion
    <Todos :todos="todos" @del-todo="deleteTodo" @edit-todo="editTodo"/>
```
If you want you can abbreviate this.